bookstore/server.js

```diff
@@ -5,10 +5,15 @@ cds.once('served', require('./srv/mashup'))
 
 // Add routes to UIs from imported packages
 cds.once('bootstrap',(app)=>{
-  app.serve ('/bookshop') .from ('@capire/bookshop','app/vue')
-  app.serve ('/reviews') .from ('@capire/reviews','app/vue')
-  app.serve ('/orders') .from('@capire/orders','app/orders')
-  app.serve ('/data') .from('@capire/data-viewer','app/viewer')
+  try {
+    app.serve ('/bookshop') .from ('@capire/bookshop','app/vue')
+    app.serve ('/reviews') .from ('@capire/reviews','app/vue')
+    app.serve ('/orders') .from('@capire/orders','app/orders')
+    app.serve ('/data') .from('@capire/data-viewer','app/viewer')
+  } catch (err) {
+    if (err.code === 'MODULE_NOT_FOUND') throw new Error('Run "npm ci" to install the required dependencies')
```

Comment:
Should the cause be set here as well?

```suggestion
    if (err.code === 'MODULE_NOT_FOUND') throw new Error('Run "npm ci" to install the required dependencies', { cause: err })
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/cause